src/components/pages/AdminDashboard/DevTools/DayComponents/01_Sat-Mon.js

```diff
@@ -74,16 +93,15 @@ const SatMon = ({ setDate }) => {
           </p>
           <p>Select the game state you would like to see in play:</p>
           <div className="state-buttons">
-            <Radio.Group>
-              <Radio className="radio-buttons">User has read</Radio>
-              <Radio className="radio-buttons">User has read and drawn</Radio>
-              <Radio className="radio-buttons">
-                User has read, drawn, and written
-              </Radio>
+            <Radio.Group onChange={onChange} value={value}>
+              <Radio value={0}>has completed 0 tasks</Radio>
+              <Radio value={1}>hasRead</Radio>
+              <Radio value={2}>hasRead and hasDrawn</Radio>
+              <Radio value={3}>hasRead, hasDrawn, and hasWritten</Radio>
             </Radio.Group>
             <Button
               className="simulate-button"
-              onClick={handleSim}
+              onClick={onSimulate}
```

Comment:
I think we could just call handleGetChildTasks here, since all onSimulate does is calls handleGetChildTasks

Reply:
We're on the same wavelength - I already did it, just haven't pushed the changes yet